cl/api/templates/rest-docs-vlatest.html

```diff
@@ -501,7 +522,7 @@ <h3 id="idb-data-endpoint">{% url 'fjcintegrateddatabase-list' version='v3' %}</
     </p>
 
 
-    <h3 id="docket-entry-endpoint">{% url "docketentry-list" version="v3" %} <span class="alt">&amp;</span> {% url "recapdocument-list" version="v3" %}</h3>
+    <h3 id="docket-entry-endpoint">{% url "docketentry-list" version="v3" %} &amp; {% url "recapdocument-list" version="v3" %}</h3>
```

Comment:
Not a fan of fancy ampersands eh?

Reply:
They were inconsistent. Some ampersands on the page (and in the menu) were fancy, and others were not. If you like fancy ampersands, I'll be happy to make them _all_ fancy, I just didn't like the mixture.

Reply:
It's all good. :)